requirements.txt

```diff
@@ -4,4 +4,5 @@ uvicorn[standard]==0.20.0
 google-cloud-bigquery==3.4.1
 pandas==1.5.2
 numpy==1.24.0
-types-python-dateutil
\ No newline at end of file
+types-python-dateutil
+types-requests
```

Comment:
Actually "types" dependencies should go into `requirements.dev.txt` I think because they are only used by `mypy`.